align_system/algorithms/outlines_regression_adm.py

```diff
@@ -40,60 +40,96 @@ def get_chain_of_thought_reasoning(target_kdma, action, state, choice, label):
+    elif target_kdma['kdma'] == 'QualityOfLife':
+        if action_character is not None:
+            # Better quality of life patient
+            if expected_value > 5:
+                cot_reasoning += f"Because {action_character.name} would have better quality of life post-treatment"
+                # Sub-optimal treatment
+                if expected_value == 7:
+                    cot_reasoning += f" but it is not the best treatment approach"
+                cot_reasoning += f", choosing "
+            # Worse quality of life patient
+            elif expected_value < 5:
+                cot_reasoning += f"Because {action_character.name} would have worse quality of life post-treatment"
+                # Sub-optimal treatment
+                if expected_value == 3:
+                    cot_reasoning += " but the supplies needed to treat the other casualty would still be available"
+                cot_reasoning += ", choosing "
+
+    # Percieved quanity of lives saved
+    # Example:
+        # Because Casualty G would likely survive treatment, choosing Treat Casualty G with hemostatic gauze would score a 9 for the the principle of perceived quantity of lives saved.
+        # Because Casualty G would likely survive treatment but it is not the best treatment approach, choosing Treat Casualty G with pressure bandage would score a 7 for the the principle of perceived quantity of lives saved.
+        # Because Casualty H would likely not survive treatment but the supplies needed to treat the other casualty would still be available, choosing Treat Casualty H with pressure bandage would score a 3 for the the principle of perceived quantity of lives saved.        
+        # Because Casualty H would likely not survive treatment, choosing Treat Casualty H with hemostatic gauze would score a 1 for the the principle of perceived quantity of lives saved.
+    elif target_kdma['kdma'] == 'PerceivedQuantityOfLivesSaved':
+        if action_character is not None:
+            # Better vol patient
+            if expected_value > 5:
+                cot_reasoning += f"Because {action_character.name} would likely survive treatment"
+                # Sub-optimal treatment
+                if expected_value == 7:
+                    cot_reasoning += f" but it is not the best treatment approach"
+                cot_reasoning += f", choosing "
+            # Worse quality of life patient
+            elif expected_value < 5:
+                cot_reasoning += f"Because {action_character.name} would likely not survive treatment"
+                # Sub-optimal treatment
+                if expected_value == 3:
+                    cot_reasoning += " but the supplies needed to treat the other casualty would still be available"
+                cot_reasoning += ", choosing "
 
     else:
         cot_reasoning = f"Choosing "
+        
     cot_reasoning += f"{choice} would score a {expected_value} for the the principle of {target_kdma['name']}."
```

Comment:
I think this is an unintended unindent. Instead of appending, you may just want to continue the string on the next line with `\`

Reply:
The unindent is intentional but may be unclear, so I fixed here: https://github.com/ITM-Kitware/align-system/pull/96/commits/e0d0bbc2589d77949a02944ba7c3292db9297a76

If the choice is associated with a character, then the cot will be like: "Because {character} is... choosing, {choice} would score a {expected_value} for the the principle of {target_kdma['name']}"
If there is no character, then it will just be: "Choosing {choice} would score a {expected_value} for the the principle of {target_kdma['name']}"
Either way the reasoning will always end with "{choice} would score a ..."